cmd/qosserver/main_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var server, client *net.UDPConn
+var err error
```

Comment:
can we have this variable inside test units? It's pretty uncommon for an error variable to have package level visibility